src/ceph_fuse.cc

```diff
@@ -204,7 +204,8 @@ int main(int argc, const char **argv, const char *envp[]) {
           "client_try_dentry_invalidate");
 	bool can_invalidate_dentries =
           client_try_dentry_invalidate && ver < KERNEL_VERSION(3, 18, 0);
-	int tr = client->test_dentry_handling(can_invalidate_dentries);
+	bool should_abort_on_failure = false;
+	int tr = client->test_dentry_handling(can_invalidate_dentries, should_abort_on_failure);
```

Comment:
nit: you could probably drop the "should_" from the name, and make it a little shorter.

Reply:
Done